features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -227,6 +227,7 @@
 
   "authentication.authenticator.office365.name": "Office365Authenticator",
   "authentication.authenticator.office365.parameters.Tags": "OIDC",
+  "authentication.authenticator.office365.parameters.UseOrgSpecificCommonAuthURL": false,
```

Comment:
shall we keep default value as true as that is the correct value which aligns with other authenticators(OIDC)?